src/test/groovy/com/onesignal/androidsdk/MainTest.groovy

```diff
@@ -825,6 +825,71 @@ class MainTest extends Specification {
         }
     }
 
+    def 'fcm 15 and crashlytics with firebase platform BOM'() {
+        when:
+        def results = runGradleProject([
+            'android.useAndroidX': true,
```

Comment:
Does `'android.enableJetifier': true,` need to be added as well since Android X is `true`?

Reply:
Good question but no, this test does not have any libraries in it also using the Android Support library.